FhirToDataLake/src/Microsoft.Health.Fhir.Synapse.Common/Extensions/ConfigurationRegistrationExtensions.cs

```diff
@@ -114,9 +114,64 @@ private static void ValidateConfiguration(this IServiceCollection services)
                 .GetRequiredService<IOptions<SchemaConfiguration>>()
                 .Value;
```

Comment:
There is bool configuration in schemaConfiguration, it is better to add try catch as other configurations to handle the case the users provide invalid value for it, such as string type value.